### PR TITLE
issue-55: node 6 compatibility

### PR DESCRIPTION
**Pull request by oliverfoster**
https://github.com/adaptlearning/adapt-cli/issues/55

tested working in node 6.9.1 and 4.5.0

to test:

use v4 or v6 of node

([download](https://github.com/adaptlearning/adapt-cli/archive/issue-55.zip) / git clone)
```
npm install
./bin/adapt create course
```

### CODE REVIEW

**Comment by hrajotia** [comment]
Nice one Ollie 👍 
could you add this to package.json https://github.com/adaptlearning/adapt-cli/pull/45/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R46

**Comment by oliverfoster** [comment]
sorry @hrajotia 
i didn't see you'd already done this. what a waste of effort! sorry dude

**Comment by oliverfoster** [comment]
how important is the travis.yml stuff?

**Comment by hrajotia** [comment]
No worries Ollie... mine was way old.. we need a fresh one... 👍 

travis.yml stuff seems to me not required.. previously node-gyp installation was broken that is why I have included them.

I think we can borrow these from authoring tool.. https://github.com/adaptlearning/adapt_authoring/blob/develop/.travis.yml#L3-L6 (excluding the v5)

File: package.json [hunk 1]
```diff
@@ -7,40 +7,41 @@
...
   "devDependencies": {
     "expect.js": "~0.3.1",
-    "grunt": "~0.4.5",
-    "grunt-mocha-test": "~0.12.7",
-    "matchdep": "~0.3.0",
-    "mocha": "~2.2.1",
-    "mockery": "~1.4.0",
-    "rimraf": "~2.3.2",
-    "sinon": "~1.14.1"
+    "grunt": "~1.0.1",
```
Comment (hrajotia) [inline package.json]: I think we can safely remove this as `grunt` already exists in dependencies object.

**Comment by hrajotia** [comment]
Excellent Ollie.. 👍

**Comment by oliverfoster** [comment]
@hrajotia when this goes in, and gets published, would you be comfortable if i deleted adaptlearning/download ?

**Comment by hrajotia** [comment]
@oliverfoster sure.. i think it have served its purpose.. 👍

**Comment by aniketdharia** [comment]
+1

**Comment by tomgreenfield** [comment]
Getting some issues with `npm install`:

**Pull request merged**